### DISCUSSION TITLE
Can you submit to the root.tsx's loader function?

### DISCUSSION THREAD

**Discussion by geepara** [remix-run/remix#6141] [Q&A]
The navbar for my app is located in root.tsx. My loader function in root.tsx returns some data that the navbar needs in order to update after a user sign-in. Is there any way I can trigger this loader function? Maybe something along the lines of `submit(null, { action: "root", method: "GET" })` ?

**Comment by sergiodxa** [comment]
If the user signing in is done with an action the loaders will re run automatically.

If not you can use the useRevalidator hook to trigger all loaders to run again.

**Reply by geepara** [reply]
Hmm I couldn't get useRevalidator to run the root loader again. I'm going to restructure the app so that this is not an issue. Thanks for the help anyways!
